pkg/graphql/client.go

```diff
@@ -68,8 +69,10 @@ func (c *ClientImpl) ExecuteQuery(ctx context.Context, query string, variables m
 
 	res, err := c.httpClient.Do(req)
 	if err != nil {
-		if urlErr, ok := err.(*url.Error); ok {
-			if x509Err, ok := urlErr.Err.(x509.UnknownAuthorityError); ok {
+		var urlErr *url.Error
+		if errors.As(err, &urlErr) {
+			var x509Err x509.UnknownAuthorityError
```

Comment:
I think this should be a pointer type.

```suggestion
			var x509Err *x509.UnknownAuthorityError
```

Also, I don't think this functionality should be in the GraphQL client, as its unrelated to it, but rather it should be in the installation info package. 

https://github.com/giantswarm/kubectl-gs/blob/6d9b79edd21404cd23fe3e54d2bcb076af891d3d/pkg/installation/info.go#L45